src/elements/Button/Button.styl

```diff
@@ -1,8 +1,19 @@
 @import "~@/assets/tokens/tokens.styl"
-
 @import "~@/styles/colors.styl"
 @import "~@/styles/font-reset.styl"
 @import "~@/styles/sizes.styl"
+@import "~@/styles/control.styl"
+@import "~@/patterns/LicenseBadge/LicenseBadge.styl"
+
+
+ // Logo Assign CC BY-SA
```

Comment:
What does this line mean?

Reply:
I wanted to explain that what follows regards only the logo from the LicenseBadge style page. It can be removed as well.

Reply:
```suggestion

```